src/test/passwordRest.spec.js

```diff
@@ -0,0 +1,83 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import TestHelper from '../utils/testHelper';
+import db from '../models';
+
+chai.use(chaiHttp);
+chai.should();
+
+
+describe('/api/v1/auth', () => {
+  let passwordResetToken;
+
+  before((done) => {
+    TestHelper.destroyModel('User');
+    done();
+  });
+  describe('POST /forgot_password', () => {
```

Comment:
leave a line after line 18

Reply:
Eslint flags it when I try to leave a space, should I disable it?